tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsWriter.cs

```diff
@@ -252,8 +289,9 @@ private void HandleConfigUpdate(AgentConfiguration config)
             }
             else
             {
-                Log.Warning("Data streams monitoring was enabled but is not supported by the Agent. Disabling Data streams. " +
-                            "Consider upgrading your Datadog Agent to at least version 7.34.0+");
+                Log.Warning(
+                    "Data streams monitoring was enabled but is not supported by the Agent. Disabling Data streams. " +
+                    "Consider upgrading your Datadog Agent to at least version 7.34.0+");
```

Comment:
> Consider upgrading your Datadog Agent to at least version 7.34.0+

Mentioning the agent version doesn't work in serverless where we use different agents each one with its own versioning scheme.

Reply:
I think it's ok to just remove that last part, "Consider upgrading your Datadog Agent. ~to at least version 7.34.0+~".